src/clj/rems/application/model.clj

```diff
@@ -167,7 +178,7 @@
              :application/licenses (map (fn [license]
                                           {:license/id (:license/id license)})
                                         (:application/licenses event))
-             :application/accepted-licenses {"applicant" #{}}
+             :application/accepted-licenses {}
```

Comment:
Why did this change?

Reply:
This was completely wrong. There should not be accepted licenses for the fixed value called "applicant" for every application. There should only be the correct ones that come later.